app/src/main/java/com/expidev/gcmapp/MainActivity.java

```diff
@@ -257,6 +262,18 @@ void onLoadChurches(@Nullable final List<Church> churches) {
         updateMap(false);
     }
 
+    void onLoadAssignment(@Nullable final Assignment assignment)
```

Comment:
was your only usage of the CurrentAssignmentLoader in MainActivity to set `PREF_CURRENT_ASSIGNMENT`?